web/templates/navbar.hbs

```diff
@@ -55,7 +55,9 @@
             </div>
             <div id="personal-menu" class="hidden-for-spectators navbar-item">
                 <a class="header-button tippy-zulip-delayed-tooltip" tabindex="0" role="button" data-tooltip-template-id="personal-menu-tooltip-template">
-                    <img class="header-button-avatar" src="{{user_avatar}}"/>
+                    <div class="header-button-avatar">
+                        <img src="{{user_avatar}}"/>
```

Comment:
please add a class on this; it's more readable that way. `header-button-avatar-image` will do OK.